benchmarks/rigid_shear/time-dependent/run_all_models

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Run all models in this directory. Note that resolution 8 and ppd 96 requires 192-384 GB of RAM
```

Comment:
But the biggest number in the list is ppd 80, not 96 and the resolution just goes up to 6? So maybe say instead what the largest numbers are that can be run on a laptop?